fixtures/esm/build-test-esm/src/index.js

```diff
@@ -0,0 +1 @@
+export default 'exported as default';
```

Comment:
Any reason why this export is different to the others?  Can we make them all 'exported as default'?